### DISCUSSION TITLE
How does the Custodian user role work?

### DISCUSSION THREAD

**Discussion by Natii9898** [globaleaks/globaleaks-whistleblowing-software#3937] [Q&A] [answered]
Good afternoon,

Since the 4.14.0 update I see that there are two new roles for users. One is that of an analyst who already more or less intuits what it is about.

But the other one called Custodian, I can't figure out what it's for. I was searching for information on the internet and couldn't find anything about it.
Could someone tell me what it is for, and how it can be used?

Thank you

**Comment by evilaliv3** [comment, marked as answer]
The custodian user role is a role implemented for the Italian National Authority for Anticorruption.

A custodian is user that do not have access to reports and do not have access to the identity of whistleblowers, but that could authorize a recipient to access the identity of a whistleblower on the base of a motivated request.

It works as following:
- Suppose to have in the system: A whistleblower, A Recipient and A custodian and to have configured questionnaire using the "Whistleblower Identity" question present inside the templates of questions.
- Suppose the whistleblower complete the question providing their details
- The recipient will be able to ask access to the identity of the whistleblower to the custodian by expressing a motivation for the request
- The custodian will be able to authorize or negate the request by providing a motivation for the denial.

**Reply by Natii9898** [reply]
Thank you very much for the clarification.

**Comment by coccolesto** [comment]
hello,
we are an italian PA and would like to use glk with the custodian role.
We created an user with the custodian role and sent a test report  with the identity of the whistleblower, but the RPCT received it directly with the identity. We have looked at the docs but we have not found  how to set up correctly the custodian role for authorizing the RCPT to request and read the  whistleblower's identity.

**Reply by evilaliv3** [reply]
Hello @coccolesto,

Thank you for your question.
I've update the answer above detailing how the function works.

The documentation about this feature is not complete because rarely and having to balance the use of our resources we have still not fond ways of proceeding on this one.
